---
layout: page
permalink: /publications/
title: Publications and Preprints
---

<i>

<ul>
<b> 1.Presented paper on a National Seminar “Sustainable Rice production Technology for Increasing the farmer’s income” titled - “Hydrological Modelling of Mahanadi Basin and their tributary exist in Chhattisgarh and part of Odisha”</b>, dated 20-21 Jan’2018 Organized by Indira Gandhi Agricultural University, Raipur (C.G.), India.
<b>Authors: Pragya Das, P.V. Raju and Chilka Sharma 9th National Seminar IGKVV, India.</b>
Change in crop sown in a region based on the water balance component trend analysis for a particular region, LULC can be managed efficiently.
	<a href="https://krishikosh.egranth.ac.in/handle/1/5810049155?mode=full"><div class="color-button">pdf</div></a><a href="https://krishikosh.egranth.ac.in/handle/1/5810049155?mode=full"><div class="color-button">cite</div></a>
	<br>
	<br>
<b> 2.Chapter in "Projection of future precipitation over Gangotri glacier at Himalayan belt using CMIP5 Climate Model : 2021"
Chapter 23 ”Advances in Hydrology and Climate Change: Historical Trends and New Approaches in Water Resources Management”.</b><br>
Published by Taylor Francis Group<br> 
</b>Authors: S.K. Chandniha, M. Arora, L. Pal, A. Kumar, P. Das, G.K. Das</b>
Climatic variability and its behavior is a complex phenomenon that is directly associated with uncertainties.In the climate change study, particularly in hydrological aspects, it is necessary to identify the parameters (predictors) that are directly or indirectly associated with predictands. The forecasted results are directly associated with the selection of predictors.
 <br>
	 <a href="../pdfs/TbasCO_preprint.pdf"><div class="color-button">pdf</div></a><a href="https://www.biorxiv.org/content/10.1101/2021.12.04.471239v1"><div class="color-button">cite</div></a><a href="https://github.com/Jorisvansteenbrugge/TbasCO"><div class="color-button">software</div></a><a href="https://github.com/elizabethmcd/EBPR-MAGs"><div class="color-button">code</div></a><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA714686"><div class="color-button">data</div></a><a href="https://figshare.com/projects/EBPR_Trait-Based_Comparative_Omics/90437"><div class="color-button">supplementary data</div></a>
	<br>
	<br>
<b> 10. Microbial cysteine production is a source of hydrogen sulfide in oxic freshwater lakes. </b> Tran P.Q.#, Bachand S.C#., Hotvedt J.C.#, Kieft K., <b><u> McDaniel E.A. </u></b>, McMahon K.D., Anantharaman K. <i> bioRxiv. Submitted. </i> Dec. 2021. DOI: 10.1101/2021.11.30.467465 <br>
	<a href="../pdfs/Tran2021_preprint.pdf"><div class="color-button">pdf</div></a><a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=OJFxBWMAAAAJ&citation_for_view=OJFxBWMAAAAJ:LkGwnXOMwfcC"><div class="color-button">cite</div></a>
	<br>
	<br>
<b> 9. Re-evaluation of the phylogenetic diversity and global distribution of the lineage Candidatus Accumulibacter. </b> Petriglieri F., Singleton C.M., Kondrotaite Z., Dueholm M.S., <b><u> McDaniel E.A. </u></b>, McMahon K.D., Nielsen P.H. <i> mSystems. </i> April 2022. DOI: 10.1128/msystems.00016-22. <br>
	<a href="../pdfs/Petriglieri2022_msystems.00016-22.pdf"><div class="color-button">pdf</div></a><a href="https://journals.asm.org/doi/10.1128/msystems.00016-22"><div class="color-button">cite</div></a>
	<br>
	<br>
<b>8. Prospects for Multi-omics in the Microbial Ecology of Water Engineering. <u>McDaniel E.A.</u></b>, Wahl S.A., Ishii S., Pinto A., Ziels R., Nielsen P.H., McMahon K.D., Williams R.B.H. <i>Water Research.</i> Oct. 2021. DOI: 10.1016/j.waterres.2021.117608 <br>
		<a href="../pdfs/McDaniel2021_WaterResearch_publication.pdf"><div class="color-button">pdf</div></a><a href="https://www.sciencedirect.com/science/article/abs/pii/S0043135421008034?via%3Dihub"><div class="color-button">cite</div></a>
	<br>
	<br>
<b>7. Metabolic differentiation of co-occurring Accumulibacter clades revealed through genome-resolved metatranscriptomics. <u>McDaniel E.A.#,</u></b> Moya-Flores F.#, Keene Beach N., Camejo P.Y., Oyserman B.O., Kizaric M., Khor E.H., 
Noguera D.R., McMahon K.D. <i>mSystems.</i> July 2021, 6 (4) e00474-21; DOI: 10.1128/mSystems.00474-21 <br>
		<a href="../pdfs/mSystems.00474-21.pdf"><div class="color-button">pdf</div></a><a href="https://journals.asm.org/doi/10.1128/mSystems.00474-21"><div class="color-button">cite</div></a><a href="https://github.com/elizabethmcd/R3R4"><div class="color-button">code</div></a><a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA668760"><div class="color-button">data</div></a><a href="https://figshare.com/projects/Metabolic_Plasticity_of_Accumulibacter_Clades/90614"><div class="color-button">supplementary data</div></a>
	<br>
	<br>
<b>6. Genome-Resolved Metagenomics of a Photosynthetic Bioreactor Performing Biological Nutrient Removal. <u>McDaniel E.A.,</u></b> Wever R., Oyserman B.O., Noguera D.R., McMahon K.D. <i>Microbiology Resource Announcements.</i> May 2021, 10 (18) e00244-21; DOI: 10.1128/MRA.00244-21  <br>
		<a href="../pdfs/mra2021_POB_publication.pdf"><div class="color-button">pdf</div></a><a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=OJFxBWMAAAAJ&citation_for_view=OJFxBWMAAAAJ:W7OEmFMy1HYC"><div class="color-button">cite</div></a><a href="https://github.com/elizabethmcd/obscurePOS"><div class="color-button">code</div></a><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA704939"><div class="color-button">data</div></a><a href="https://figshare.com/projects/POB_Genome-Resolved_Metagenomics/99704"><div class="color-button">supplementary data</div></a>
	<br>
	<br>
<b>5. Community-led, integrated, reproducible multi-omics with anvi’o.</b> Eren A.M., Kiefl E., Shaiber A., Veseli I., Miller S.E., Schecter M.S., Fink I., Pan J.N., Yousef M., Fogarty E.C., Trigodet F., Watson A.R., Esen O.C., Moore R.M., Clayssen Q., Lee M.D., Kivenson V., Graham E.D., Merrill B.D., Karkman A., Blankenberg D., Eppley J.M., Sjoden A., Scott J.J., Vazquez-Campos X., McKay L.J, <u><b> McDaniel E.A.,</b></u> Stevens S.L.R., Anderson R., Fuessel J., Fernandez-Guerra A., Maignien L., Delmont T.O., Willis A.D.  <i>Nature Microbiology (Commentary).</i> Jan. 2021, 6, 3-6. DOI: 10.1038/s41564-020-00834-3.  <br>
		<a href="../pdfs/Anvio2020_NMC.pdf"><div class="color-button">pdf</div></a><a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=OJFxBWMAAAAJ&citation_for_view=OJFxBWMAAAAJ:Y0pCki6q_DkC"><div class="color-button">cite</div></a>
	<br>
	<br>
	<b>4. Mercury methylation genes identified across diverse anaerobic microbial guilds in a eutrophic sulfate-enriched lake.</b> Peterson B.D., <u><b>McDaniel E.A.,</b></u> Schmidt A.G., Lepak R.F., Tran P.Q., Marick R.A., Ogorek J.M., DeWild J.F., Krabbenhoft D.P., McMahon K.D.  <i>Environmental Science and Technology.</i> Nov. 2020, 54 (24), 15840-15851; DOI: 10.1021/acs.est.0c05435 <br>
		<a href="../pdfs/Peterson2020_EST_finalPub.pdf"><div class="color-button">pdf</div></a><a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=OJFxBWMAAAAJ&citation_for_view=OJFxBWMAAAAJ:zYLM7Y9cAGgC"><div class="color-button">cite</div></a><a href="https://osf.io/9vwgt/"><div class="color-button">data</div></a>	
	<br>
	<br>
	<b>3. Expanded Phylogenetic Diversity and Metabolic Flexibility of Mercury-Methylating Microorganisms.<u> McDaniel E.A.,</u> </b> Peterson B., Stevens S.L.R., Tran P.Q., Anantharaman K., McMahon K.D. <i>mSystems.</i> Aug 2020, 5 (4) e00299-20; DOI: 10.1128/mSystems.00299-20 <br>
		<a href="../pdfs/mSystems-2020-McDaniel-e00299-20.full.pdf"><div class="color-button">pdf</div></a><a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=OJFxBWMAAAAJ&citation_for_view=OJFxBWMAAAAJ:IjCSPb-OGe4C"><div class="color-button">cite</div></a><a href="https://github.com/elizabethmcd/MEHG"><div class="color-button">code</div></a><a href="https://figshare.com/projects/Expanded_Diversity_and_Metabolic_Flexibility_of_Microbial_Mercury_Methylation/70361"><div class="color-button">data</div></a>	
	<br>
	<br>
	<b>2. metabolisHMM: Phylogenomic analysis for exploration of microbial phylogenies and metabolic pathways. <u>McDaniel E.A.,</u></b> Anantharaman, K., McMahon K.D. <i>bioRxiv.</i> Dec. 2019, DOI: 10.1101/2019.12.20.884627. <br>
		<a href="../pdfs/2020-01-09-metabolisHMM-manuscript-preprint-corrected-submission.pdf"><div class="color-button">pdf</div></a><a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=OJFxBWMAAAAJ&citation_for_view=OJFxBWMAAAAJ:9yKSN-GCB0IC"><div class="color-button">cite</div></a><a href="https://github.com/elizabethmcd/metabolisHMM"><div class="color-button">software</div></a>
	<br>
	<br>
<b>1. Independent Mechanisms for Acquired Salt Tolerance versus Growth Resumption Induced by Mild Ethanol Pretreatment in Saccharomyces cerevisiae. <u>McDaniel E.A.,</u></b> Stuecker T.N., Veluvolu M., Gasch A.P., Lewis J.A. <i>mSphere. Editors Pick. </i> Nov 2018, 3 (6) e0057418; DOI: 10.1128/mSphere.00574-18 <br>
		<a href="../pdfs/mSphere-2018-McDaniel-e00574-18.full.pdf"><div class="color-button">pdf</div></a><a href="https://scholar.google.com/citations?view_op=view_citation&hl=en&user=OJFxBWMAAAAJ&citation_for_view=OJFxBWMAAAAJ:d1gkVwhDpl0C"><div class="color-button">cite</div>
	<br>
	<br>
